components/rule-mgt/org.wso2.carbon.identity.rule.management/src/main/java/org/wso2/carbon/identity/rule/management/dao/impl/RuleData.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
```

Comment:
Year shouldn't change (assuming it was commited in 2024). Either it should be 2024 or or a range of years.

Reply:
Moving the classes updates the year by default. Not sure on best practice. I didn't change as the IDE applies that.
It makes sense to keep the same year as the code is committed before

Reply:
Fixed